tests/loggers/test_tensorboard.py

```diff
@@ -25,12 +25,10 @@
 from pytorch_lightning import Trainer
 from pytorch_lightning.loggers import TensorBoardLogger
 from tests.helpers import BoringModel
+from tests.helpers.runif import RunIf
 
 
-@pytest.mark.skipif(
-    LooseVersion(torch.__version__) < LooseVersion("1.5.0"),
-    reason="Minimal PT version is set to 1.5",
-)
+@RunIf(min_torch="1.5")
```

Comment:
You should use the full version, see why:

```
LooseVersion("1.5.0") == LooseVersion("1.5")
Out[13]: False
```

Reply:
good point
```python
LooseVersion("1.5.0") < LooseVersion("1.5")
Out[7]: False
LooseVersion("1.5.0") > LooseVersion("1.5")
Out[8]: True
```